crowbar_framework/app/models/backup.rb

```diff
@@ -23,6 +23,9 @@ class Backup < ActiveRecord::Base
   before_validation :save_or_create_archive, on: :create
   after_validation :delete_archive, unless: -> { errors.empty? }
   after_destroy :delete_archive
+  after_create do
+    data.rmtree
```

Comment:
This is not needed, Rails will delete the tmpdir when the instance is collected by the garbage collector.